Enable snapping of windows by default.

This patch by Thierry Reding.
Not submitted to upstream.

Index: compiz.git/plugins/compiz.schemas.in
===================================================================
--- compiz.git.orig/plugins/compiz.schemas.in	2006-12-31 16:44:56.000000000 +0100
+++ compiz.git/plugins/compiz.schemas.in	2006-12-31 18:44:44.000000000 +0100
@@ -12775,7 +12775,7 @@
             <applyto>/apps/compiz/plugins/wobbly/allscreens/options/snap_inverted</applyto>
             <owner>compiz</owner>
             <type>bool</type>
-            <default>false</default>
+            <default>true</default>
             <locale name="C">
                 <short>Snap Inverted</short>
                 <long>Inverted window snapping</long>
Index: compiz.git/plugins/wobbly.c
===================================================================
--- compiz.git.orig/plugins/wobbly.c	2006-12-31 16:44:56.000000000 +0100
+++ compiz.git/plugins/wobbly.c	2006-12-31 18:44:44.000000000 +0100
@@ -168,7 +168,7 @@
 
 #define WOBBLY_SNAP_MODIFIERS_DEFAULT ShiftMask
 
-#define WOBBLY_SNAP_INVERTED_DEFAULT FALSE
+#define WOBBLY_SNAP_INVERTED_DEFAULT TRUE
 
 #define WOBBLY_MAXIMIZE_EFFECT_DEFAULT TRUE
 
